src/Microsoft.Windows.CsWin32/Generator.cs

```diff
@@ -3885,6 +3885,12 @@ private IEnumerable<MemberDeclarationSyntax> CreateCommonTypeDefMembers(Identifi
         // If this typedef struct represents a pointer, add an IsNull property.
         if (fieldType is IdentifierNameSyntax { Identifier: { Value: nameof(IntPtr) or nameof(UIntPtr) } })
         {
+            // public static readonly IntPtr Null;
+            yield return FieldDeclaration(VariableDeclaration(structName.WithTrailingTrivia(TriviaList(Space)))
+                .AddVariables(VariableDeclarator(IdentifierName("Null").Identifier)))
+                .AddModifiers(TokenWithSpace(SyntaxKind.PublicKeyword), TokenWithSpace(SyntaxKind.StaticKeyword), TokenWithSpace(SyntaxKind.ReadOnlyKeyword)) // operators MUST be public
```

Comment:
There's a comment talking about operator declarations on code for a field declaration.